CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Change Log for MIGraphX
+
+Full documentation for MIGraphX is available at [MIGraphX Github Pages](https://rocmsoftwareplatform.github.io/AMDMIGraphX/doc/html/).
+
+## MIGraphX 2.5 for ROCm 5.5.0
+
+### Added
```

Comment:
Should we mention Dynamic Ref ops ?

Reply:
I don't really consider Ref to be something people would use us for.  We will announce Dynamic shapes when it is read, but I think the ref portion was really about getting a solid test foundation.